File: ejercicio6.cpp
#include <iostream>

using namespace std;

int main()
{
    int x=100;
	for(int i=0;i<10;i++)
	{
	    cout<<"Global: "<<x<<endl;
	    int x=i;
	    cout<<"Local: "<<x<<endl;
	}
	
	return 0;
}

File: ejercicio6.cs
using System;
 
public class Test
{
	public static void Main()
	{
		int x=100;
		for(int i=0;i<10;i++)
		{
		    Console.WriteLine("Global: "+x);
		    int x=i;
		    Console.WriteLine("Local: "+x);
		}
	}
}